Policy tags docs (#946)
dataform-co/dataform: content/docs/warehouses/bigquery.md

select
  *
from
  ${ref("raw_events")}
where event_timestamp > event_timestamp_checkpoint
```

This will avoid a full table scan on the `raw_events` table when inserting new rows, only looking at the most recent partitions it needs to.

## Managing BigQuery policy tags

<callout intent="warning">
  Support for managing policy tags was introduced from Dataform version <code>1.8.6</code>.
</callout>

> If you use [policy tags](https://cloud.google.com/bigquery/docs/column-level-security-intro) for managing column-level security in BigQuery, then you can set policy tags on columns in tables via the Dataform config block. Note that any policy tags manually applied to datasets created via Dataform will likely get overwritten as the table is updated so you'll want to make sure that you configure them in Dataform instead.

Reviewer: ```suggestion
If you use [policy tags](https://cloud.google.com/bigquery/docs/column-level-security-intro) for managing column-level security in BigQuery, then you can set policy tags on columns in tables via the Dataform config block. Note that any policy tags created directly in BigQuery will get overwritten when Dataform updates the table, so make sure that you configure all policy tags within Dataform.
```
